torch_geometric/distributed/partition.py

```diff
@@ -167,9 +175,9 @@ def generate_partition(self):
                         data.edge_index[:, global_eid],
                         torch.stack((global_row, global_col), dim=0),
                     )
-                    local_eid = global_eid - edge_offset[edge_type]
```

Comment:
I think it's more clear if we keep the original naming because `global_eid` becomes `local_eid` within edge group.

Reply:
These are still global edge ids but belonging to a specific partition. If we would keep the original naming we will write to edge_feats:
{
'global_id': local_eid,
...
}
which I found a bit misleading. Therefore, I suggest changing the name to, for example: partition_eids? What do You think?

Reply:
@JakubPietrakIntel After some thought, I changed the name to "offseted_eid", because from what I noticed, the introduction of offset is the reason for changing the name to local_eid.